pkgs/tools/misc/changelogger/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, buildDate ? "1970-01-01"
+}:
+
+buildGoModule rec {
+  pname = "changelogger";
+  version = "0.5.2";
+
+  src = fetchFromGitHub {
+    owner = "MarkusFreitag";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-r/rnYutjETlE05A/kVQEQ6ICtWXW6kMMfPrv1rw5f3Q=";
+  };
+
+  vendorSha256 = "sha256-RmLSuLZdYpA557xN7fkPZm5ektxvRHil1E2u1qR7EO0=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/MarkusFreitag/changelogger/cmd.BuildVersion=${version}"
+    "-X github.com/MarkusFreitag/changelogger/cmd.BuildDate=${buildDate}"
```

Please specify the date here. It should be `1970-01-01T00:00:00` because the upstream uses gorelaser which follows this template.
```suggestion
    "-X github.com/MarkusFreitag/changelogger/cmd.BuildDate=1970-01-01T00:00:00"
```